### PR TITLE
Update direct-routing-sbc-multiple-tenants.md

### PR DESCRIPTION
**Pull request by NMURAV**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 1]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
 
 ### Create a trunk and provision users
 
-> [!NOTE]
-> Based on feedback we received in the Technical Adoption Program, Microsoft might change the process of creating trunks in the customer tenants to simplify the process. Please watch the documentation updates on this page and follow the Microsoft Technical Community blogs for further information. 
-
-Create a trunk in the customer domain using the New-CSonlinePSTNGateway command. The trunk FQDN **MUST** match the subdomain created for the customer.
+With the initial release of the Direct Routing, Microsoft required to add a trunk to each served tenant (customer tenant) using New-CSOnlinePSTNGateway.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
With the initial release of Direct Routing, Microsoft required a trunk to be added to each served tenant (customer tenant) using New-CSOnlinePSTNGateway.
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 2]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
 
-For example:
+However, this showed as not being optimal for two reasons:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
However, this proved to be not being optimal for two reasons:
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 3]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+ 
+• Management overhead. For example, offloading or draining an SBC, changing some parameters, like enabling or disabling media bypass, changing the port required changing parameters in multiple tenants (by running Set-CSonlinePSTNGateway), but in fact, it is the same SBC;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
• **Overhead management**. Offloading or draining an SBC, for example, changes some parameters, like enabling or disabling media bypass. Changing the port requires changing parameters in multiple tenants (by running Set-CSonlinePSTNGateway), but it is in fact the same SBC; 
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 4]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
-> Based on feedback we received in the Technical Adoption Program, Microsoft might change the process of creating trunks in the customer tenants to simplify the process. Please watch the documentation updates on this page and follow the Microsoft Technical Community blogs for further information. 
-
-Create a trunk in the customer domain using the New-CSonlinePSTNGateway command. The trunk FQDN **MUST** match the subdomain created for the customer.
+With the initial release of the Direct Routing, Microsoft required to add a trunk to each served tenant (customer tenant) using New-CSOnlinePSTNGateway.
 
-For example:
+However, this showed as not being optimal for two reasons:
+ 
+• Management overhead. For example, offloading or draining an SBC, changing some parameters, like enabling or disabling media bypass, changing the port required changing parameters in multiple tenants (by running Set-CSonlinePSTNGateway), but in fact, it is the same SBC; 
+• Processing overhead. Gathering and monitoring trunk health data - SIP Options collected from multiple logical trunks but in reality, it is the same SBC and the same physical trunk, which slows down processing of the data for routing
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
• **Overhead processing**. Gathering and monitoring trunk health data - SIP options collected from multiple logical trunks that are, in reality, the same SBC and the same physical trunk, slows down processing of the routing data.
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 5]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+ 
 
-```
-New-CSOnlinePSTNGateway –FQDN sbc1.customers.adatum.biz -SipSignallingPort 5068
-```
+Based on this feedback, Microsoft is bringing a new logic to provision the trunks for the customer tenants.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
Based on this feedback, Microsoft is bringing in a new logic to provision the trunks for the customer tenants.
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 6]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
 
-When creating the trunk, you may receive the following error message:
+Two new entities introduced:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
Two new entities introduced:
```
```suggestion
Two new entities are introduced:
```

Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
Two new entities are introduced:
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 7]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+•	Carrier trunk registered in carrier tenant, using the command New-CSOnlinePSTNGateway, for example
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	Carrier trunk registered in the carrier tenant using the command New-CSOnlinePSTNGateway, for example New-CSOnlinePSTNGateway -FQDN customers.adatum.biz -SIPSignallingport 5068 -ForwardPAI $true.
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 8]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+New-CSOnlinePSTNGateway -FQDN customers.adatum.biz -SIPSignallingport 5068 -ForwardPAI $true
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion

```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 9]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+•	Derived trunk does not require registration. It is simple a desired host name added in from of the carrier trunk. It derives all configuration parameters from the carrier trunk. The derived trunk doesn't need to be created in PowerShell, the association with the carrier trunk based on the FQDN name (see details below);
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	Derived trunk, which does not require registration. It is simply a desired host name added in from of the carrier trunk. It derives all its configuration parameters from the carrier trunk. The derived trunk doesn't need to be created in PowerShell, and the association with the carrier trunk is based on the FQDN name (see details below).
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 10]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
 
-```
-Can not use the "sbc1.customers.adatum.biz" domain as it was not configured for this tenant.
-```
+Provisioning logic and example.
 
-Please allow some time for domain registration and activation to replicate and try again.
+•	Carriers only need to set up and manage a single trunk  (carrier trunk in the carrier domain), using Set-CSOnlinePSTNGateway command. In the example above it is adatum.biz;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	Carriers only need to set up and manage a single trunk  (carrier trunk in the carrier domain), using the Set-CSOnlinePSTNGateway command. In the example above it is adatum.biz;
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 11]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+•	In customer tenant, the carrier need only to add the derived trunk FQDN to voice routing policies of the users. There is no need to run New-CSOnlinePSTNGateway for a trunk.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	In customer tenant, the carrier need only to add the derived trunk FQDN to voice routing policies of the users. There is no need to run New-CSOnlinePSTNGateway for a trunk.
```
```suggestion
•	In the customer tenant, the carrier need only to add the derived trunk FQDN to the voice routing policies of the users. There is no need to run New-CSOnlinePSTNGateway for a trunk.
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 12]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+•	 The derived trunk as the name suggests inherits or derives all the configuration parameters from the carrier trunk.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	 The derived trunk, as the name suggests, inherits or derives all the configuration parameters from the carrier trunk. 
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 13]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
-Can not use the "sbc1.customers.adatum.biz" domain as it was not configured for this tenant.
-```
+Provisioning logic and example.
 
-Please allow some time for domain registration and activation to replicate and try again.
+•	Carriers only need to set up and manage a single trunk  (carrier trunk in the carrier domain), using Set-CSOnlinePSTNGateway command. In the example above it is adatum.biz;
+•	In customer tenant, the carrier need only to add the derived trunk FQDN to voice routing policies of the users. There is no need to run New-CSOnlinePSTNGateway for a trunk.
+•	 The derived trunk as the name suggests inherits or derives all the configuration parameters from the carrier trunk. 
+Examples:
+•	Customers.adatum.biz – carrier trunk which needs to be created in the carrier tenant
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	Customers.adatum.biz – the carrier trunk which needs to be created in the carrier tenant.
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 14]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+•	Sbc1.customers.adatum.biz – derived trunk in a customer tenant, that don’t need to be created i PowerShell.  You can simply add in Online Voice Routing Policy the name of derived trunk in the customer tenant without creating it.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	Sbc1.customers.adatum.biz – the derived trunk in a customer tenant that does not need to be created in PowerShell.  You can simply add the name of the derived trunk in the customer tenant in the online voice routing policy without creating it.
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 15]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
 
-Provision users with the phone numbers and configure voice routing.
+•	All changes made on carrier trunk (on carrier tenant) automatically applied on derived trunks. For example, carriers can change a SIP port on the carrier trunk, and this change applies to all derived trunks. New logic to configure the trunks simplifies the management as you don’t need to go to every tenant and change the parameter on every trunk;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	Any changes made on a carrier trunk (on carrier tenant) is automatically applied to derived trunks. For example, carriers can change an SIP port on the carrier trunk, and this change applies to all derived trunks. New logic to configure the trunks simplifies the management as you don’t need to go to every tenant and change the parameter on every trunk.
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 16]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+•	The Options sent only to carrier trunk FQDN. The health status of the carrier trunk applied to all derived trunks. Note the health status is used for routing decisions, you can read about how Direct Routing uses Options here https://docs.microsoft.com/en-us/microsoftteams/direct-routing-monitor-and-troubleshoot
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	The options are sent only to the carrier trunk FQDN. The health status of the carrier trunk is applied to all derived trunks and is used for routing decisions. To find out more about Direct Routing options, refer to [Monitor and troubleshoot Direct Routing] https://docs.microsoft.com/en-us/microsoftteams/direct-routing-monitor-and-troubleshoot.
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 17]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+•	Carrier can drain the carrier trunk; all derived trunks will be drained as well.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	The carrier can drain the carrier trunk, in which case all derived trunks will be drained as well. 
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 18]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
 
-Provision users with the phone numbers and configure voice routing.
+•	All changes made on carrier trunk (on carrier tenant) automatically applied on derived trunks. For example, carriers can change a SIP port on the carrier trunk, and this change applies to all derived trunks. New logic to configure the trunks simplifies the management as you don’t need to go to every tenant and change the parameter on every trunk;
+•	The Options sent only to carrier trunk FQDN. The health status of the carrier trunk applied to all derived trunks. Note the health status is used for routing decisions, you can read about how Direct Routing uses Options here https://docs.microsoft.com/en-us/microsoftteams/direct-routing-monitor-and-troubleshoot
+•	Carrier can drain the carrier trunk; all derived trunks will be drained as well. 
+ 
 
-For more information on the New-CSOnlinePSTNGateway, provisioning users, and configuring voice routing, please refer to [Configure Direct Routing](direct-routing-configure.md).
+Migration from the previous model to the Carrier Trunk
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
Migration from the previous model to the carrier trunk
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 19]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
+ 
+For migration from the current implementation of the carrier hosted model to the new model, the carriers will need to reconfigure the trunks for customers. Remove the trunks from customer tenants, using Remove-CSOnluinePSTNGateway (leaving the trunk in the carrier tenant).
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
For migration from the current implementation of the carrier hosted model to the new model, the carriers will need to reconfigure the trunks for customer tenants. Remove the trunks from the customer tenants using Remove-CSOnluinePSTNGateway (leaving the trunk in the carrier tenant).
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 20]
```diff
@@ -195,29 +195,41 @@ For example: test@sbc1.customers.adatum.biz
...
 
+We highly encourage migrating to the new solution as soon as possible as we will be enhancing the monitoring and provisioning using the carrier and derived trunk model.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
We highly encourage migrating to the new solution as soon as possible as we will be enhancing monitoring and provisioning using the carrier and derived trunk model.
```

**Comment by NMURAV** [comment]
How can we move this forward?

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 21]
```diff
@@ -195,29 +195,40 @@ For example: test@sbc1.customers.adatum.biz
...
 ### Create a trunk and provision users
 
-> [!NOTE]
-> Based on feedback we received in the Technical Adoption Program, Microsoft might change the process of creating trunks in the customer tenants to simplify the process. Please watch the documentation updates on this page and follow the Microsoft Technical Community blogs for further information. 
-
-Create a trunk in the customer domain using the New-CSonlinePSTNGateway command. The trunk FQDN **MUST** match the subdomain created for the customer.
+With the initial release of Direct Routing, Microsoft required a trunk to be added to each served tenant (customer tenant) using New-CSOnlinePSTNGateway.
 
-For example:
+However, this has proved to be not optimal for two reasons:
```
Comment (nenonix) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
However, this has not proved optimal for two reasons:
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 22]
```diff
@@ -195,29 +195,40 @@ For example: test@sbc1.customers.adatum.biz
...
+Examples:
+•	Customers.adatum.biz – the carrier trunk which needs to be created in the carrier tenant.
+•	Sbc1.customers.adatum.biz – the derived trunk in a customer tenant that does not need to be created in PowerShell.  You can simply add the name of the derived trunk in the customer tenant in the online voice routing policy without creating it.
 
-Provision users with the phone numbers and configure voice routing.
+•	Any changes made on a carrier trunk (on carrier tenant) is automatically applied to derived trunks. For example, carriers can change an SIP port on the carrier trunk, and this change applies to all derived trunks. New logic to configure the trunks simplifies the management as you don’t need to go to every tenant and change the parameter on every trunk.
+•	The options are sent only to the carrier trunk FQDN. The health status of the carrier trunk is applied to all derived trunks and is used for routing decisions. To find out more about Direct Routing options, refer to [Monitor and troubleshoot Direct Routing] https://docs.microsoft.com/en-us/microsoftteams/direct-routing-monitor-and-troubleshoot.
```
Comment (nenonix) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	The options are sent only to the carrier trunk FQDN. The health status of the carrier trunk is applied to all derived trunks and is used for routing decisions. Find out more about [Direct Routing options](https://docs.microsoft.com/microsoftteams/direct-routing-monitor-and-troubleshoot).
```

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 23]
```diff
@@ -195,29 +195,40 @@ For example: test@sbc1.customers.adatum.biz
...
+•	The carrier can drain the carrier trunk, in which case all derived trunks will be drained as well.
```
Comment (nenonix) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	The carrier can drain the carrier trunk, and all derived trunks will be drained as well. 
```

**Comment by scanum** [comment]
@CarolynRowe Changes look correct. Could you please review and merge? Thanks!

**Pull request merged**